Form/FormFlowInterface.php

```diff
@@ -27,9 +27,9 @@ function getName();
 	function setFormFactory(FormFactoryInterface $formFactory);
 
 	/**
-	 * @param Request|null $request
+	 * @param RequestStack $requestStack
 	 */
-	function setRequest(Request $request = null);
+	function setRequestStack(RequestStack $requestStack);
```

Comment:
I could remove it from the interface, but don't like constructor injection, prefer setters.